openreview/stages/venue_stages.py

```diff
@@ -1062,6 +1064,59 @@ def get_invitees(self, conference, number):
 
         return invitees
 
+    def get_chat_invitees(self, conference, number):
+        invitees = [conference.get_id(), conference.support_user]
+
+        if conference.use_senior_area_chairs and self.Readers.SENIOR_AREA_CHAIRS_ASSIGNED in self.invitees:
+            invitees.append(conference.get_senior_area_chairs_id(number))
+
+        if conference.use_area_chairs and self.Readers.AREA_CHAIRS_ASSIGNED in self.invitees:
+            invitees.append(conference.get_area_chairs_id(number))
+
+        if self.Readers.REVIEWERS_ASSIGNED in self.invitees:
+            invitees.append(conference.get_reviewers_id(number))
+
+        if self.Readers.REVIEWERS_SUBMITTED in self.invitees:
```

Comment:
I don't think we need both Reviewers and Reviewers/Submitted right?

Reply:
it depends on which participants they choose in the CommentStage.

Reply:
but Reviewers/Submitted could be an option.

Reply:
OK, it makes sense that Reviewers/Submitted could be an option, but both Reviewers and Reviewers/Submitted shouldn't be in readers at the same time. It's confusing in the UI if it says the chat is visible to Reviewers and Reviewers/Submitted